d3rlpy/logging/wandb_adapter.py

```diff
@@ -0,0 +1,73 @@
+import wandb
+from typing import Any, Dict, Optional
+from .logger import LoggerAdapter, LoggerAdapterFactory, SaveProtocol
+
+
+class LoggerWanDBAdapter(LoggerAdapter):
+    r"""WandB Logger Adapter class.
+
+    This class logs data to Weights & Biases (WandB) for experiment tracking.
+
+    Args:
+        experiment_name (str): Name of the experiment.
+    """
+
+    def __init__(self, project: Optional[str] = None, experiment_name: Optional[str] = None):
+        self.run = wandb.init(project=project, name=experiment_name)
```

Comment:
This is just a question. I'm not familiar with wandb, but if `project` is `None`, what would happen?

Reply:
If project is `None`, wandb will save this run in a new project, whose name is inferred from the relative directory the script resides at. E.g. if you start a training script from `scripts/training/train_sac.py` the WanDB project will be called `REPO_NAME-scripts_training_train_sac`, where REPO_NAME is the folder name of the git repository